Dockerfile

```diff
@@ -22,6 +24,17 @@ RUN echo "if [ -f ${WS}/install/setup.zsh ]; then source ${WS}/install/setup.zsh
 RUN echo 'eval "$(register-python-argcomplete3 ros2)"' >> $HOME/.zshrc
 RUN echo 'eval "$(register-python-argcomplete3 colcon)"' >> $HOME/.zshrc
 RUN echo "if [ -f ${WS}/install/setup.bash ]; then source ${WS}/install/setup.bash; fi" >> $HOME/.bashrc
+
+RUN --mount=type=secret,id=GITHUB_RAE_PAT_TOKEN if [ "$INCLUDE_SPECTACULARAI_ROS" = "YES" ]; then \
+      rm -rf sai_ros \
+      && git clone --single-branch --branch ${SPECTACULARAI_ROS_VERSION} https://github.com/SpectacularAI/ros.git sai_ros \
+      && cd sai_ros \
+      && apt-get -y install unzip --no-install-recommends \
+      && ROS_DISTRO=$ROS_DISTRO DEPTHAI_WS=$UNDERLAY_WS GITHUB_RAE_PAT_TOKEN=$(cat /run/secrets/GITHUB_RAE_PAT_TOKEN) . ./scripts/download_and_build_static.sh \
```

Comment:
Change `/run/secrets/GITHUB_RAE_PAT_TOKEN` to `/run/secrets/SPECTACULAR_AI_TOKEN`